obs-packaging/versions.txt

```diff
@@ -13,7 +13,7 @@ kata_proxy_version=1.1.0
 kata_shim_version=1.1.0
 kata_agent_version=1.1.0
 kata_ksm_throttler_version=1.1.0
-kata_osbuilder_version=1.1.0
+kata_osbuilder_version=master
```

Comment:
This is just for the sake of getting ppc64le related changes post 1.1.0 release. If all other aspects of the patch are ok, then this should be merged with `1.2.0` version of kata_osbuilder.